src/index.html

```diff
@@ -9,5 +9,53 @@
   </head>
   <body>
     <h1>Stars</h1>
+    <div class="stars--0">
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+      <div class="stars--star"></div>
+    </div>
```

Comment:
use proper BEM
stars is block name,  stars--0 is modifier and stars__star is an element

```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```